forge-gui/res/cardsfolder/upcoming/monster_manual_zoological_study.txt

```diff
@@ -0,0 +1,16 @@
+Name:Monster Manual
+ManaCost:3 G
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 1 G T | Origin$ Hand | Destination$ Battlefield | ChangeType$ Creature | ChangeNum$ 1 | SpellDescription$ You may put a creature card from your hand onto the battlefield.
+AlternateMode:Adventure
+Oracle:{1}{G}, {T}: You may put a creature card from your hand onto the battlefield.
+
+ALTERNATE
+
+Name:Zoological Study
+ManaCost:2 G
+Types:Sorcery Adventure
+A:SP$ Mill | Cost$ 2 G | Defined$ You | NumCards$ 5 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill five cards, then return a creature card milled this way to your hand.
```

Comment:
Again, Cost isn't needed.

"You" is actually the default for all `Defined$` params that look at players ("Self" is the default for `Defined$` params that look at cards) so you don't need that either.